lld/ELF/Driver.cpp

```diff
@@ -459,6 +460,8 @@ static void checkOptions() {
       error("-z force-bti only supported on AArch64");
     if (config->zBtiReport != "none")
       error("-z bti-report only supported on AArch64");
+    if (config->zPauthReport != "none")
+      error("-z pauth-report only supported on AArch64");
```

Comment:
untested

Reply:
Thanks, fixed in 589c6455a929f41ed3a79fc7d91119586eb1ee7b